### DISCUSSION TITLE
active waypoint action  missing #2 Programming Framework.md

### DISCUSSION THREAD

**Discussion by synersignart** [iNavFlight/inav#8678] [General]
<html xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

guys I have a question hope it isnt a typo but on the  active waypoint action the missing number 2 . is this correct its in the documentation Programming Framework.md

![image](https://user-images.githubusercontent.com/22503442/210468620-759fb8f5-48c3-49e2-a1af-b0508a30e920.png)

<meta name=ProgId content=PowerPoint.Slide>
<meta name=Generator content="Microsoft PowerPoint 15">
<style>

</style>
</head>

<body>

Action | Value
-- | --
WAYPOINT | 1
HOLD_TIME | 3
RTH | 4
SET_POI | 5
JUMP | 6
SET_HEAD | 7
LAND | 8

</body>

</html>

**Comment by stronnag** [comment]
It is correct. See the [canonical WP documentation](https://github.com/iNavFlight/inav/wiki/MSP-Navigation-Messages).
